### DISCUSSION TITLE
gcp Deployment

### DISCUSSION THREAD

**Discussion by beekboss** [serverpod/serverpod#2877] [Q&A] [answered]
I deployed project to gcp , step by step as docs , deployment success , database created , but I got errors:

1-  Error: googleapi: Error 403: Another user owns the domain storage.domain.app or a parent domain. You can either verify domain ownership at https://search.google.com/search-console/welcome?new_domain_name=storage.domain.app or find the current owner and ask that person to create the bucket for you, forbidden
2- Error: googleapi: Error 403: Another user owns the domain private-storage.domain.app or a parent domain. You can either verify domain ownership at https://search.google.com/search-console/welcome?new_domain_name=private-storage.awazem.app or find the current owner and ask that person to create the bucket for you, forbidden

3-  no such file or directory: /deploy/gcp/terraform_gce/.terraform/modules/serverpod_production ( deployment failed , I had to remove this directory then deployment success )

4- flutter: SocketException: Failed host lookup: 'api.domain.app' (OS Error: nodename nor servname provided, or not known, errno = 8)
when try to run app.

any help please, thanks

**Comment by SandPod** [comment]
Hey @beekboss!

> 1, 2 and 4

Did you follow the guide for deploying with Terraform? https://docs.serverpod.dev/deployments/deploying-to-gce-terraform

It seems like you have not verified ownership for the domain you are trying to use. Validate that you have done this for all domains you are using: https://docs.serverpod.dev/deployments/deploying-to-gce-terraform#verify-your-domain

The top domain that will be used is configured in your servers `deploy/gcp/terraform_gce/config.auto.tfvars` file.

**Reply by beekboss** [reply]
I already verified the domain ownership
and already follow the guid step by step

**Reply by SandPod** [reply]
>  Error: googleapi: Error 403: Another user owns the domain storage.domain.app or a parent domain. You can either verify domain ownership at https://search.google.com/search-console/welcome?new_domain_name=storage.domain.app or find the current owner and ask that person to create the bucket for you, forbidden

Suggests that the verification was not successful. Are you using the same account to deploy Serverpod and validate the domain? What happens if you visit the linked address?

**Reply by beekboss** [reply]
yes , using same account to deploy and validate , if I visit link This site can’t be reached.

**Reply by beekboss** [reply]
domain is "awazem.app"

**Reply by beekboss** [reply]
zone name is awazem, dns name is awazem.app , which one have to use in terraform config?

**Reply by SandPod** [reply, marked as answer]
Try to configure your top domain to `awazem.app`.

The domain in the link suggests that you are trying to use the domain `domain.app` as your top domain.

**Comment by beekboss** [comment]
thanks , success deployment
